client/src/components/habits/Habits/Habit.tsx

```diff
@@ -10,7 +12,10 @@ type HabitProps = {
 
 export default function Habit({ habit }: HabitProps) {
 	// const { mutate } = useHabitDeleteMutation();
-	// const { openDetailedHabitModal } = useDetailedHabitModal();
+	const { openDetailedItemModal } = useDetailedItemModal(
```

Comment:
Comment this out for now. Want to implement a button that triggers the modal to open. Think I have an open todo for that somewhere, so ignore this comment for this PR.